src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -0,0 +1,101 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import { Link } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { makeCromwellStatusLine } from 'src/components/job-common'
+import { FlexTable, tableHeight, TooltipCell } from 'src/components/table'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+import { FailuresModal } from 'src/pages/workspaces/workspace/jobHistory/FailuresViewer'
+
+
+const CallTable = ({ namespace, name, submissionId, workflowId, callName, callObjects }) => {
+  const [failuresModalParams, setFailuresModalParams] = useState()
+
+  return div([
+    h(AutoSizer, { disableHeight: true }, [
+      ({ width }) => h(FlexTable, {
+        height: tableHeight({ actualRows: callObjects.length, maxRows: 10 }),
```

Comment:
On a Mac notebook it was not immediately obvious to me that this table is scrollable.

This is because scroll bars are hidden by default on devices with trackpads, and the cells are perfectly aligned to their container.

If the container displayed e.g. 10.5 cells it would be more obvious that the table continues below.

![Screenshot_2021-02-08 cromwell-tests-workspace - Workflow Dashboard](https://user-images.githubusercontent.com/1087943/107275976-6de2a600-6a20-11eb-9451-902cd20ebc61.png)

Reply:
I wonder whether @panentheos has thoughts on this. I was also tempted to have an extra half-row teaser to clue people in that there was more content below... but it felt kind of hacky. Is this a standard thing to do?

Reply:
That actually sounds like a pretty good solution. There isn't a universal pattern for hinting that a table is scrollable. Some apps use shading tricks to accomplish that, but since we don't do that, adding an extra half row might be a good way to handle it.